project/Build.scala

```diff
@@ -1382,6 +1384,8 @@ object Build {
       generateReferenceDocumentation := Def.inputTaskDyn {
         val shouldRegenerateExpectedLinks = literal("--no-regenerate-expected-links").?.parsed.isEmpty
 
+        staticAssets.value
```

Comment:
This won't work since we are copying entire `docs` directory before running `generateDocumentation`. The static assets need to be present in `docs` before copy